### PR TITLE
Icon category

### PR DESCRIPTION
**Pull request by daosgc**
- Show icon category and update styles
- Add new param icon category in edit and new category form in admin module
- Fix routing to edit a category in admin module

### CODE REVIEW

**Comment by daosgc** [comment]
Admin Module
Screen Shot 2020-07-25 at 10 38 12

**Comment by daosgc** [comment]
Client Module
Screen Shot 2020-07-25 at 10 38 18

File: src/app/shared-pages/categories/edit-category/edit-category.component.html [hunk 1]
```diff
@@ -1,3 +1,12 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button></ion-back-button>
+    </ion-buttons>
+    <ion-title>Editar</ion-title>
```
Comment (fabricioflores) [inline src/app/shared-pages/categories/edit-category/edit-category.component.html:6]:
```suggestion
    <ion-title>Editar categoría</ion-title>
```

File: src/app/shared-pages/categories/category-form/category-form.component.html [hunk 2]
```diff
@@ -7,6 +7,13 @@
     <app-control-messages [control]="categoryFormGroup.get('name')">
     </app-control-messages>
 
+    <ion-item>
+      <ion-label position="stacked">Icono<ion-text color="danger">*</ion-text></ion-label>
```
Comment (fabricioflores) [inline src/app/shared-pages/categories/category-form/category-form.component.html:11]:
```suggestion
      <ion-label position="stacked">Ícono<ion-text color="danger">*</ion-text></ion-label>
```

File: src/app/shared-pages/categories/new-category/new-category.component.html [hunk 3]
```diff
@@ -1,3 +1,12 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button></ion-back-button>
+    </ion-buttons>
+    <ion-title>Editar</ion-title>
```
Comment (fabricioflores) [inline src/app/shared-pages/categories/new-category/new-category.component.html:6]:
```suggestion
    <ion-title>Nueva categoría</ion-title>
```

**Pull request merged**